catalog/spatial/geocoding/spatial-geocoding-lucene/src/test/java/org/codice/ddf/spatial/geocoding/query/TestGeoNamesQueryLuceneIndex.java

```diff
@@ -166,16 +165,16 @@ private Document createDocumentFromGeoEntry(final GeoEntry geoEntry) {
         document.add(new TextField(GeoNamesLuceneConstants.NAME_FIELD,
                 geoEntry.getName(),
                 Field.Store.YES));
-        document.add(new DoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
+        document.add(new LegacyDoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
                 geoEntry.getLatitude(),
                 Field.Store.YES));
-        document.add(new DoubleField(GeoNamesLuceneConstants.LONGITUDE_FIELD,
+        document.add(new LegacyDoubleField(GeoNamesLuceneConstants.LONGITUDE_FIELD,
                 geoEntry.getLongitude(),
                 Field.Store.YES));
         document.add(new StringField(GeoNamesLuceneConstants.FEATURE_CODE_FIELD,
                 geoEntry.getFeatureCode(),
                 Field.Store.YES));
-        document.add(new LongField(GeoNamesLuceneConstants.POPULATION_FIELD,
+        document.add(new LegacyLongField(GeoNamesLuceneConstants.POPULATION_FIELD,
```

Comment:
I looked at LongPoint, but its ctor does take the Field.Store.YES argument. I didn't know how that would change the behavior.